src/library/basetracktablemodel.cpp

```diff
@@ -416,7 +416,9 @@ QAbstractItemDelegate* BaseTrackTableModel::delegateForColumn(
                 &BaseTrackTableModel::slotRefreshCoverRows);
         return pCoverArtDelegate;
     }
-    return nullptr;
+    // Explicitly return a TableItemDelegate, otherwise
+    // the default QStyleItemDelegate will be used.
+    return new TableItemDelegate(pTableView);
```

Comment:
The ownership is not obvious here. No issue of this PR though. 
setItemDelegateForColumn() does not take the ownership: https://doc.qt.io/qt-6/qabstractitemview.html#setItemDelegateForColumn
Can we introduce a parented pointer?